js/remove_string_spaces.js

```diff
@@ -0,0 +1,12 @@
+// https://www.codewars.com/kata/remove-string-spaces/train/javascript
```

Comment:
What's with the formatting in this file? Is prettier turned off?

How about adding the pre-commit hook as I suggested earlier 😄 ?

Reply:
Issue added